src/ontology/mondo-edit.obo

```diff
@@ -153524,7 +153524,6 @@ xref: OMIM:132090 {source="Orphanet:25968", source="MONDO:equivalentTo", source=
 xref: Orphanet:25968 {source="MONDO:equivalentTo", source="OMIM:132090"}
 xref: UMLS:C1851549 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:377052"}
 is_a: MONDO:0015650 {source="https://orcid.org/0000-0001-5208-3432"} ! epilepsy syndrome
-is_a: MONDO:0020072 {source="https://orcid.org/0000-0001-5208-3432", source="https://www.epilepsydiagnosis.org/syndrome/epilepsy-syndrome-groupoverview.html"} ! childhood-onset epilepsy syndrome
 relationship: excluded_subClassOf MONDO:0020072 {source="Orphanet:25968", source="https://orcid.org/0000-0001-5208-3432"} ! childhood-onset epilepsy syndrome
```

Comment:
@nicolevasilevsky can you please review this? 
Is this the correct source of action? I don't know which came first, the SC or the "excluded_SubclassOf"?

Reply:
FWIW, in cases where the Subclass Sync pipeline added parents that were also mentioned in an `excluded_subClassOf` I removed manually since the pipeline does not handle this yet

Reply:
I think we should remove the excluded_subClassOf axiom, the is_a assertion looks legitimate

Reply:
@twhetzel Can you please make this change? I can review and merge afterwards.

Reply:
FYI - there's an update to the Subclass Sync. I need to see how this PR and that one (https://github.com/monarch-initiative/mondo/pull/8431) are related and whether this one (#8422) should be closed and the pipeline re-run.

Reply:
Ok, I'll make the change....

Reply:
done

Reply:
Nico [said](https://github.com/monarch-initiative/mondo/pull/8431#issuecomment-2492588395) to merge this PR here

Reply:
> Nico https://github.com/monarch-initiative/mondo/pull/8431#issuecomment-2492588395 to merge this PR here
I don't understand what that means. But the PR is approved

Reply:
Thanks, it means this PR (#8422) should still be merged despite another [PR](https://github.com/monarch-initiative/mondo/pull/8431#issuecomment-2492588395) related to the Subclass Sync he also pushed.

I will merge #8422 - thanks!